domain/policy.go

```diff
@@ -182,15 +182,30 @@ type Requirement struct {
 
 // Policy is the approval policy configuration
 type Policy struct {
-	ID           string            `json:"id" yaml:"id" validate:"required"`
-	Version      uint              `json:"version" yaml:"version" validate:"required"`
-	Description  string            `json:"description" yaml:"description"`
-	Steps        []*Step           `json:"steps" yaml:"steps" validate:"required,min=1,dive"`
-	Requirements []*Requirement    `json:"requirements,omitempty" yaml:"requirements,omitempty" validate:"omitempty,min=1,dive"`
-	Labels       map[string]string `json:"labels,omitempty" yaml:"labels,omitempty"`
-	IAM          *IAMConfig        `json:"iam,omitempty" yaml:"iam,omitempty" validate:"omitempty,dive"`
-	CreatedAt    time.Time         `json:"created_at,omitempty" yaml:"created_at,omitempty"`
-	UpdatedAt    time.Time         `json:"updated_at,omitempty" yaml:"updated_at,omitempty"`
+	ID           string              `json:"id" yaml:"id" validate:"required"`
+	Version      uint                `json:"version" yaml:"version" validate:"required"`
+	Description  string              `json:"description" yaml:"description"`
+	Steps        []*Step             `json:"steps" yaml:"steps" validate:"required,min=1,dive"`
+	AppealConfig *PolicyAppealConfig `json:"appeal_config" yaml:"appeal_config" validate:"omitempty"`
```

Comment:
```suggestion
	AppealConfig *PolicyAppealConfig `json:"appeal_config" yaml:"appeal_config" validate:"omitempty,dive"`
```
CMIIW but I think `dive` is needed if we want to validate AppealConfig struct as well when validating the Policy struct.